src/createApolloClient.js

```diff
@@ -26,9 +28,19 @@ const middlewareLink = setContext(() => {
   };
 });
 
+// Create a RestLink for the REST API
+// If you are using multiple link types, restLink should go before httpLink,
+// as httpLink will swallow any calls that should be routed through rest!
+const restLink = new RestLink({
+  endpoints: {
+    cfa: 'https://api.codeforamerica.org/api/'
+  },
+  credentials: 'same-origin'
```

Comment:
Wouldn't we just want to say `omit` here? We aren't sending any credentials to the Code for America API